src/overlays/actors/ovl_En_Xc/z_en_xc.c

```diff
@@ -226,6 +226,7 @@ void func_80B3C7D4(EnXc* this, s32 action1, s32 action2, s32 action3) {
     }
 }
 
+#if OOT_DEBUG
 s32 EnXc_NoCutscenePlaying(PlayState* play) {
     if (play->csCtx.state == 0) {
         return true;
```

Comment:
this should be using the cs state enum. yet another thing to cleanup at some point.
as usual, dont have to do this now if you dont want